samples/Directory.Packages.props

```diff
@@ -1,54 +1,54 @@
-		<PackageVersion Include="Uno.Extensions.Logging.OSLog" Version="1.3.0" />
-		<PackageVersion Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.4.0" />
-		<PackageVersion Include="Uno.Material" Version="2.7.0-dev.97" />
-		<PackageVersion Include="Uno.Material.WinUI" Version="2.7.0-dev.97" />
-		<PackageVersion Include="Uno.UI" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.5.9" />
-		<PackageVersion Include="Uno.UI.RemoteControl" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.UI.Skia.Gtk" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.UI.Skia.Tizen" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.UI.Skia.Wpf" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.UI.WebAssembly" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.UniversalImageLoader" Version="1.9.35" />
-		<PackageVersion Include="Uno.Wasm.Bootstrap" Version="7.0.3" />
-		<PackageVersion Include="Uno.Wasm.Bootstrap.DevServer" Version="7.0.3" />
-		<PackageVersion Include="Uno.WinUI" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.WinUI.RemoteControl" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.WinUI.Skia.Gtk" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Uno.WinUI.WebAssembly" Version="4.10.0-dev.85" />
-		<PackageVersion Include="Xamarin.Android.Support.CustomTabs" Version="28.0.0.3" />
-		<PackageVersion Include="Xamarin.AndroidX.Browser" Version="1.4.0.2" />
-		<PackageVersion Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.3.1.1" />
-		<PackageVersion Include="Xamarin.Jetbrains.Annotations" Version="23.0.0.4" />
-		<PackageVersion Include="Xamarin.TestCloud.Agent" Version="0.23.0" />
-		<PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="4.5.0-2.final" />
-		<PackageVersion Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0-preview1.22518.1" />
-	</ItemGroup>
-</Project>
+    <PackageVersion Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.12" />
+    <PackageVersion Include="Microsoft.UI.Xaml" Version="2.8.5" />
+    <PackageVersion Include="Microsoft.Windows.Compatibility" Version="5.0.1" />
+    <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.1.0" />
+    <PackageVersion Include="SkiaSharp.Skottie" Version="2.88.2" />
+    <PackageVersion Include="SkiaSharp.Views" Version="2.80.2" />
+    <PackageVersion Include="SkiaSharp.Views.Uno" Version="2.88.2" />
+    <PackageVersion Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.2" />
+    <PackageVersion Include="Uno.Core.Extensions.Compatibility" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
+    <PackageVersion Include="Uno.Cupertino" Version="2.7.0-dev.97" />
+    <PackageVersion Include="Uno.Cupertino.WinUI" Version="2.7.0-dev.97" />
```

Comment:
Please keep Uno.Cupertino in sync with Uno.Material

Reply:
done